2nd-gen/alerts-to-discord/functions/index.js

```diff
@@ -139,3 +142,64 @@ UDID **${testerDeviceIdentifier}** for ${testerDeviceModelName}
   }
 });
 // [END v2Alerts]
+
+/**
+ * Function triggered by Firebase Performance that publishes
+ * a message to Discord whenever a performance threshold alert is fired.
+ */
+// [START v2PerformanceAlertTrigger]
+exports.postperformancealerttodiscord = onThresholdAlertPublished(
+    async (event) => {
+      // [END v2PerformanceAlertTrigger]
+      // [START v2PerformanceEventPayload]
+      // construct a helpful message to send to Discord
+      const appId = event.appId;
+      const {
+        eventName,
+        metricType,
+        eventType,
+        numSamples,
+        thresholdValue,
+        thresholdUnit,
+        conditionPercentile,
+        appVersion,
+        violationValue,
+        violationUnit,
+        investigateUri,
+      } = event.data.payload;
+      const message = `
+    ⚠️ Performance Alert for ${metricType} of ${eventType}: **${eventName}** ⚠️
+    
+    App id: ${appId}
+    Alert condition: ${thresholdValue} ${thresholdUnit}
+    Percentile (if applicable): ${conditionPercentile}
+    App version (if applicable): ${appVersion}
```

Comment:
nit: consider making this conditional instead of having the "if applicable" qualifier

Reply:
I am wondering what you mean by conditional, do you mean having 4 if statements and depending on which one is true, having a different string message? I am leaning towards just keeping it simple because this is just an example for developers and they should be able to format the message however they want.

Reply:
I am going to keep this simple and not do conditional, since this is meant to just be some sample code